app/src/main/java/com/xplore/paymobile/ui/more/MoreFragment.kt

```diff
@@ -28,15 +32,29 @@ class MoreFragment : Fragment() {
         _binding = FragmentMoreBinding.inflate(inflater, container, false)
         val root: View = binding.root
 
-        val textView: TextView = binding.textMore
-        moreViewModel.text.observe(viewLifecycleOwner) {
-            textView.text = it
-        }
+        populateUI()
+
         return root
     }
 
     override fun onDestroyView() {
         super.onDestroyView()
         _binding = null
     }
+
+    private fun populateUI() {
+        binding.urlText.text = Constants.BASE_URL_SANDBOX
+        binding.publicKeyText.text = Constants.PUBLIC_KEY_SANDBOX
+        binding.apiKeyText.text = Constants.API_KEY_SANDBOX
+        binding.switchButton.setOnCheckedChangeListener(object: CompoundButton.OnCheckedChangeListener {
+            override fun onCheckedChanged(buttonView: CompoundButton?, isChecked: Boolean) {
+
+                binding.prodLabel.setTextColor(ContextCompat.getColor(context!!, if (isChecked) R.color.teal_200 else R.color.black))
```

Comment:
requireContext() could be used instead of context!!

Reply:
done